src/doc/src/reference/overriding-dependencies.md

```diff
@@ -212,7 +212,7 @@ version = "0.1.0"
 
 [dependencies]
 my-library = { git = 'https://example.com/git/my-library' }
-uuid = "1.0"
+uuid = "1.0.1"
```

Comment:
> As far as I can tell the example should say 1.0.1 not 1.0

Could you expand on why you think it should be `1.0.1`?  Its not quite sticking out to me at the moment.

I don't see a direct reference to 1.0.1 in this example/section.  Other examples use `uuid = "1.0"`.  There is one example that uses 1.0.1 but its specifically dealing with an unpublished version.